client/src/pages/WeeklyTable.js

```diff
@@ -74,12 +76,55 @@ const WeeklyTable = ({
 	refreshBooking,
 	maxDesksForDay,
 }) => {
-	const bookingsByRow = getBookingsByRow(bookings, rowsCount, week);
-	console.log({ bookingsByRow });
+	const [currentMonday, setCurrentMonday] = useState(new Date(2022, 0, 17));
+	let week = [];
+	const year = currentMonday.getFullYear();
+	const month = currentMonday.getMonth();
+	let date = currentMonday.getDate();
 
+	for (let i = 0; i < 5; i++) {
+		// this will even work if we cross over into the next month!!
+		week.push(new Date(year, month, date + i).toISOString());
+	}
+	console.log(week);
+	const bookingsByRow = getBookingsByRow(bookings, rowsCount, week);
+	console.log(bookingsByRow);
 	const bookingsByDesk = getBookingsByDesk(bookings, week);
+	// function setThisMonday() {
+
+	// }
+	function setNextMonday() {
+		setCurrentMonday((currentMonday) => {
+			return new Date(
+				currentMonday.getFullYear(),
+				currentMonday.getMonth(),
+				currentMonday.getDate() + 7
```

Comment:
I can see here the reoson of my question ;)

Reply:
:)